common/htime/time.go

```diff
@@ -115,6 +115,10 @@ func (f TimeFormatter) Format(t time.Time, layout string) string {
 			return f.ltr.FmtTimeMedium(t)
 		case "time_short":
 			return f.ltr.FmtTimeShort(t)
+		case "rfc1123":
+			return t.Format(time.RFC1123)
+		case "rfc3339":
+			return t.Format(time.RFC3339)
```

Comment:
see About ISO8601 and RFC3339 diffs, see https://ijmacd.github.io/rfc3339-iso8601/

Reply:
```go
fmt.Println(time.Now().Format("2006-01-02T15:04:05-07:00"))
// output: 2009-11-10T23:00:00+00:00
fmt.Println(time.Now().Format(time.RFC3339))
// output: 2009-11-10T23:00:00Z
```

see https://go.dev/play/p/UarnITzPT_V